i18n/locales/zh-CN/hourofcode/how-to/index.md

```diff
@@ -33,34 +31,34 @@ nav: how_to_nav
 
 <img src="/images/fit-350/group_ipad.jpg" />
 
-## 5. Start your Hour of Code off with an inspiring speaker or video
+## 5）以一段激励演讲或视频来开始编程一小时活动
```

Comment:
Interestingly enough this file lost some translations but added some too. So confused.

Reply:
Looks like there was generally some weirdness with Chinese; I re-synced after fixing some syntax issues, and a whole bunch of stuff changed. PTAL